intro-Cloud/ch_ConfigureInstances.tex

```diff
@@ -39,12 +39,12 @@ \section{The \_vm, \_vsc and \_nfs networks}\label{sec:_vm-_nfs-networks}
 Instances should use the \_vm network for communication, and the \_nfs
 network if they need access to shared file systems (see chapter
 \ref{cha:shared-file-systems}). On the other hand \_vsc network is used to
-connect to or provide VSC services via VSC network and floating IPs.
-When an instance is created in
-\gls{OpenStack} and connected to the \_vm, \_nfs or \_vsc networks, it is
-automatically assigned a fixed IP address in that network. This IP
-address is permanently associated with the instance until the instance
-is terminated.
+connect to or provide VSC services via VSC network and floating or dedicated 
+project IPs. When an instance is created in \gls{OpenStack} and connected to 
+the \_vm, \_nfs or \_vsc networks, it is automatically assigned a fixed 
+IP address in that network (the predefined project IP in case of a project). 
```

Comment:
I would remove the sentence `(the predefined project IP in case of a project)`  here to avoid confusion with the project's floating IPs (explained in the next section)